src/index.html

```diff
@@ -6,9 +6,56 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin=""
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img
+        src="./images/logo.png"
+        alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a class="nav_link is-active" href="#apple">Apple</a>
+          </li>
+          <li class="nav_item">
+            <a class="nav_link" href="#">Samsung</a>
+          </li>
+          <li class="nav_item">
+            <a class="nav_link" href="#">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav_item">
+            <a class="nav_link"
+              href="#"
+              data-qa="hover"
+            >
```

Comment:
https://mateacademy.slack.com/archives/C02CSGP1KD1/p1630943541196100?thread_ts=1630943169.192800&cid=C02CSGP1KD1
```suggestion
            <a
              class="nav_link"
              href="#"
              data-qa="hover"
            >
```